### PR TITLE
Admins accounts can no longer be deleted as button is greyed out with…

### PR DESCRIPTION
**Pull request by tomcatooo**
#43 Fix

### CODE REVIEW

File: resources/views/admin/users.blade.php [hunk 1]
```diff
@@ -48,12 +48,20 @@
...
+                                <?php if ($user->is_admin != 1){?>
                                 <form action="{{ route('admin.users.destroy', $user->id) }}" method="POST">
                                     {{ csrf_field() }}
                                     {{ method_field('DELETE') }}
 
                                     <button type="submit" class="btn btn-primary">Delete</button>
                                 </form>
+                                <?php } 
+                                else{?>
+                                <button class="btn btn-primary" title="Admin accounts can not be deleted!" type="submit" disabled>Delete</button>
```
Comment (jameshmread) [inline resources/views/admin/users.blade.php:60]: bit of a long button name?

**Review by jameshmread — DISMISSED** [review]
When fixing bugs / addressing features can you tag the issue in the description / title of the pull request? then close the issue when its merged?

**Comment by ghost** [comment]
I've reopened this as reports are not auto dispatched any more due to api changes so this is valid now, also referenced the issue in head comment.

File: resources/views/admin/users.blade.php (hunk 1 above)
Comment (ghost) [inline resources/views/admin/users.blade.php:60]: I would have the button remain the same only have the disabled flag on it

```
<button` type="submit" class="btn btn-primary" disabled>Delete</button>
```

**Review by ghost — APPROVED** [review]

**Pull request merged**